amazonia/classes/asg.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, title, vpc, template, minsize, maxsize, subnets, load_balance
         """
         super(Asg, self).__init__(vpc=vpc, title=title, template=template)
         if maxsize < minsize:
```

Comment:
I wonder if we want to move this check into the Yaml class instead of the Asg class? It makes more sense there because the yaml class is for consolidation/validation.

Happy to merge this without this particular change.... Just voicing a thought that we might want to card up :)